storage/src/adb/any/mod.rs

```diff
@@ -134,19 +134,20 @@ where
     /// inactivity floor to the location following the moved operation. This method is therefore
     /// guaranteed to raise the floor by at least one. Returns the new inactivity floor location.
     ///
+    /// TODO(https://github.com/commonwarexyz/monorepo/issues/1829): callers of this method should
```

Comment:
This should be `//` rather than displayed in the docs.

Reply:
changed to //. Thought it might be good to have this in the docs to dissuade more users, but I guess more users would likely just be us!